rails/rails_trunk/rails_benchmarks/runner

```diff
@@ -1,20 +1,10 @@
 #!/bin/bash
 set -e
 
-while ! exec 6<>/dev/tcp/${POSTGRES_PORT_5432_TCP_ADDR}/${POSTGRES_PORT_5432_TCP_PORT}; do
-  echo "$(date) - still trying to connect to Postgres server"
-  sleep 1
-done
-
-while ! exec 6<>/dev/tcp/${MYSQL_PORT_3306_TCP_ADDR}/${MYSQL_PORT_3306_TCP_PORT}; do
-  echo "$(date) - still trying to connect to Mysql server"
-  sleep 1
-done
-
-dropdb --if-exists -h "$POSTGRES_PORT_5432_TCP_ADDR" -p "$POSTGRES_PORT_5432_TCP_PORT" -U postgres rubybench
-createdb -h "$POSTGRES_PORT_5432_TCP_ADDR" -p "$POSTGRES_PORT_5432_TCP_PORT" -U postgres rubybench
-echo "DROP DATABASE IF EXISTS rubybench" | mysql -u root -h "$MYSQL_PORT_3306_TCP_ADDR" -P "MYSQL_PORT_3306_TCP_PORT"
-echo "CREATE DATABASE rubybench" | mysql -u root -h "$MYSQL_PORT_3306_TCP_ADDR" -P "MYSQL_PORT_3306_TCP_PORT"
+dropdb --if-exists -h postgres -U postgres rubybench
+createdb -h postgres -U postgres rubybench
+echo "DROP DATABASE IF EXISTS rubybench" | mysql -u root -h mysql
+echo "CREATE DATABASE rubybench" | mysql -u root -h mysql
```

Comment:
We are recreating only tables used in certain benchmark but not entire database. I have feeling that we should drop and create whole database before each bench. I am not sure if having two many tables in database affects performance even though you are not using them?